### PR TITLE
Add mapping stats and picard metrics

### LINKED ISSUE #61: Add constants for mapping stat in run element metrics
CR359 requires new metrics to be added to the reporting app.
The corresponding missing constant should be added to EGCG-Core

### PR DESCRIPTION
**Pull request by tcezard**
Small PR to add new metric names
Also test that there are no duplicates in the constants
closes #61

### CODE REVIEW

File: tests/test_constant.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from egcg_core import constants as cst
+
+
+def test_constants_uniqueness():
+    constants = [item for item in dir(cst) if not item.startswith("__")]
+    uniq_constants = set()
+    for constant in constants:
+        if constant in uniq_constants:
+            assert False
+        uniq_constants.add(constant)
```
Comment (mwhamgenomics) [inline tests/test_constant.py]: - The `assert True` below will have no effect during a test
- The `assert False` above could just be `assert constant not in uniq_constants`

**Review by mwhamgenomics — CHANGES_REQUESTED** [review]

**Review by katieemelianova — APPROVED** [review]
Approved pending Murray's requested changes

File: tests/test_constant.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from egcg_core import constants as cst
+
+
+def test_constants_uniqueness():
+    constants = [item for item in dir(cst) if not item.startswith("__")]
+    uniq_constants = set()
+    for constant in constants:
+        assert constant not in uniq_constants
```
Comment (mwhamgenomics) [inline tests/test_constant.py:8]: This will just assert that `constant` is not in an empty set, which will always be True!
Edit: fixed

**Review by mwhamgenomics — CHANGES_REQUESTED** [review]

**Review by mwhamgenomics — APPROVED** [review]
